src/schema/genre/genre.typedefs.ts

```diff
@@ -0,0 +1,19 @@
+import gql from 'graphql-tag';
+
+export const genreTypeDefs = gql`
+	type MovieInGenre {
```

Comment:
We can update this when we have the movie type from movies gql

Reply:
FWIW, I'd just exclude this entirely for when you get the movie added. This is going to confuse someone.